### PR TITLE
Documenting create condition from a chart

### PR DESCRIPTION
**Pull request by paperclypse**
I grabbed the GIF from the related what's new post. (Do we even need an image here? Would a non-animated image be better? I do like the way it draws the eye...)

I don't think we need much more than this. Keeping it short and to the point seems like the best approach here.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 1]
```diff
@@ -38,6 +38,17 @@ To create a NRQL alert condition for a policy:
...
 * Under **Select a product** click **NRQL**, and then click **Next, define thresholds**.
 
+## Create a condition from a chart [#create-chart]
+
+![Animated GIF showing how to create a NRQL alerts condition from a chart.](./images/in-context-demo.gif "Create a NRQL alerts condition from a chart.")
+<figcaption>To create a NRQL alerts condition from a chart, click the chart menu <Icon name="fe-more-horizontal"/>, then click **Create alert condition**.</figcaption>
+
+Once you've named and customized your condition, you can create a new policy for it or choose an existing one.
+
+<Callout variant="tip">
```
Comment (urbiz-nr) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:48]: Tip or caution?

**Review by urbiz-nr — APPROVED** [review]

**Comment by urbiz-nr** [comment]
HI @paperclypse, I've left a comment, up to you before you merge. Tried locally, looks great!

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:48]: I think caution is a good call here.

**Comment by paperclypse** [comment]
@urbiz-nr Mind taking another pass on this one? I realized I needed some kind of intro sentence in this new section. Thanks!

**Review by urbiz-nr — APPROVED** [review]
Looks good!

**Pull request merged**